Makefile

```diff
@@ -89,11 +89,8 @@ unit-tests:
 	@echo "Running unit tests..."
 	$(GO) test -v -count=1 ./pkg/...  -json -cover | tparse --all
 
-tests-e2e: build docker-build
-	$(GO) test -p 1 -timeout 30m -v -tags e2e ./tests/e2e/connectivity/... 
-
-tests-iperf3: build docker-build 
-	$(GO) test -p 1 -timeout 30m -v -tags e2e ./tests/e2e/iperf3/...
+tests-e2e-k8s:
```

Comment:
question: what's the rationale for removing the previous test targets? I think it is because the the e2e tests both in its suite, but wanted to confirm.

nit: 
- should test targets be declared `.PHONY` or depend on the log files generated?
- consider adding logs to `.gitignore`, as suggested by @zivnevo

Reply:
> question: what's the rationale for removing the previous test targets? I think it is because the the e2e tests both in its suite, but wanted to confirm.

Correct.

Also, the log files are no longer added to the repo tree, so no change required to `.gitignore`.